sql/core/src/test/scala/org/apache/spark/sql/SubquerySuite.scala

```diff
@@ -2699,22 +2699,17 @@ class SubquerySuite extends QueryTest
   test("SPARK-42937: Outer join with subquery in condition") {
     withSQLConf(SQLConf.ADAPTIVE_EXECUTION_ENABLED.key -> "false",
       SQLConf.WHOLESTAGE_CODEGEN_ENABLED.key -> "false") {
-      withTempView("t2") {
-        // this is the same as the view t created in beforeAll, but that gets dropped by
-        // one of the tests above
-        r.filter($"c".isNotNull && $"d".isNotNull).createOrReplaceTempView("t2")
```

Comment:
This test no longer needs to re-create `t` (here named `t2`), because `t` is now available for this test.